cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-filter.php

```diff
@@ -716,7 +729,12 @@ public function setup_hooks() {
 		$filter = $this;
 		array_map(
```

Comment:
We can use this chance to get rid of the `array_map` for looping here and use a basic loop control.